.ci/integrationTestDownstream.groovy

```diff
@@ -190,11 +194,9 @@ class IntegrationTestingParallelTaskGenerator extends DefaultParallelTaskGenerat
           ]
         def label = "${tag}-${x}-${y}"
         try{
+          saveResult(x, y, 0)
```

Comment:
Will this cause any regression?

Reply:
no, the result in case of exception/no-exception is the same, the only change is that the exception is not transformed on an error so it is directly managed by the pipeline that is better.